lib/image-stream/supported.js

```diff
@@ -56,5 +56,9 @@ module.exports = [
   {
     extension: 'raw',
     type: 'image'
+  },
+  {
+    extension: 'dmg',
+    type: 'image'
```

Comment:
That's tricky. I guess `compressed` is the way to go. In any case, these type labels are only used to show an ordered tooltip of the supported file types.